src/TaxesServiceProvider.php

```diff
@@ -24,6 +24,10 @@ public function configureTipoffPackage(TipoffPackage $package): void
                 \Tipoff\Taxes\Nova\LocationTax::class,
                 \Tipoff\Taxes\Nova\Tax::class,
             ])
+            ->hasBindings([
+                \Tipoff\Support\Contracts\Taxes\TaxRequest::class => \Tipoff\Taxes\TaxRequest::class,
+                \Tipoff\Support\Contracts\Taxes\TaxRequestItem::class => \Tipoff\Taxes\TaxRequestItem::class,
+            ])
```

Comment:
These bindings to `\Tipoff\Taxes\TaxRequest::class` and `\Tipoff\Taxes\TaxRequestItem::class` shouldn't be added without the implementation of those classes as well.  I did a quick scan of the PR history and I'm not seeing that they've been added yet.